lib/storage/src/types.rs

```diff
@@ -29,6 +30,8 @@ pub struct StorageConfig {
     pub wal: WalConfig,
     pub performance: PerformanceConfig,
     pub hnsw_index: HnswConfig,
+    #[serde(default = "default_mmap_advice")]
+    pub mmap_advice: madvise::Advice,
```

Comment:
We can also add `#[serde(skip_serializing)]` attribute to this field, to make sure it won't accidentally appear anywhere during serialization.

@generall

Reply:
Fun fact: we can remove ` Serialize, JsonSchema` from the `StorageConfig` and it will still build fine.
So i think it is fine without `skip_serializing`

Reply:
Done.